pages/cars.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <Suspense>
+    <template #default>
+      <div
+        v-for="car in list"
```

Comment:
task about cars should not be part of this PR

Reply:
Don't know why it showed up again

Reply:
because commits are in this branch, you need to remove commits that are unrelated